vars/setupCommonPipelineEnvironment.groovy

```diff
@@ -65,20 +88,53 @@ void call(Map parameters = [:]) {
     }
 }
 
-private loadConfigurationFromFile(script, String configFile) {
+private static loadConfigurationFromFile(script, String configFile) {
     if (!configFile) {
         String defaultYmlConfigFile = '.pipeline/config.yml'
         String defaultYamlConfigFile = '.pipeline/config.yaml'
-        if (fileExists(defaultYmlConfigFile)) {
+        if (script.fileExists(defaultYmlConfigFile)) {
             configFile = defaultYmlConfigFile
-        } else if (fileExists(defaultYamlConfigFile)) {
+        } else if (script.fileExists(defaultYamlConfigFile)) {
             configFile = defaultYamlConfigFile
         }
     }
 
     // A file passed to the function is not checked for existence in order to fail the pipeline.
     if (configFile) {
-        script.commonPipelineEnvironment.configuration = readYaml(file: configFile)
+        script.commonPipelineEnvironment.configuration = script.readYaml(file: configFile)
         script.commonPipelineEnvironment.configurationFile = configFile
     }
 }
+
+private static List putCustomDefaultsIntoPipelineEnv(script, List customDefaults) {
```

Comment:
Naming: From the name is was not clear to me that this method is downloading the config files and coping them into the .pipeline folder. 
I think put and env is misleading? I thought with env you mean the environment variable?

Reply:
Any how is that different from copying it from the the resources?

Reply:
"pipeline env" is sometimes used to refer to `.pipeline/`. For example, piper has a config value "EnvRootPath" which refers to this folder. Of course "put" could be replaced by "copyOrDownload", what do you think?

Reply:
Okay. Never came across that.  copyOrDownload I think is better